src/CMakeLists.txt

```diff
@@ -10,6 +10,11 @@ endif()
 
 find_package(PkgConfig REQUIRED)
 
+if(CMAKE_VERSION VERSION_LESS 3.3)
+	message(STATUS "  Version of cmake is too old, data packaging will not be used.")
+	set(DISABLE_ZIP ON)
```

Comment:
I don't think this should disable ZIP support completely. If anything, it should disable only `PACKAGE_DATA`. But even then, I would prefer that in case `PACKAGE_DATA` is enabled and Cmake is too old, it would exit with an error instead of automatically disabling `PACKAGE_DATA`. The error then would explain that either Cmake should be upgraded to 3.3+, or `PACKAGE_DATA` should be disabled manually. This is consistent with how `PACKAGE_DATA_LEANIFY` is handled.